### PR TITLE
Use the default interface for reporting network stats.

### PR DESCRIPTION
**Pull request by timstclair**
Fixes https://github.com/kubernetes/kubernetes/issues/22319

### CODE REVIEW

**Comment by dchen1107** [comment]
LGTM 

cc/ @bprashanth

File: pkg/kubelet/network/network.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package network
+
+// TODO: Consider making this value configurable.
```
Comment (vishh) [inline pkg/kubelet/network/network.go]: @thockin: Do we have to make this configurable?

**Comment by vishh** [comment]
LGTM

**Comment by ixdy** [comment]
@k8s-bot test this, issue #22435

**Pull request merged**